app/templates/home.hbs

```diff
@@ -1,83 +1,61 @@
 <div class="home-page">
```

Comment:
Ok makes sense though I do believe it is easier to read templates in this way. We can try this to see if it looks good.